src/lib/libpthread.js

```diff
@@ -978,7 +978,7 @@ var LibraryPThread = {
 
   $establishStackSpace__internal: true,
   $establishStackSpace__deps: ['$stackRestore', 'emscripten_stack_set_limits'],
-  $establishStackSpace: (pthread_ptr) => {
+  $establishStackSpace: function (pthread_ptr) {
```

Comment:
why did this  change?

Reply:
It was the easy way to add another `node.id.name` check to skip the `establishStackSpace` helper too, as we skip specific function declarations.

I could instead add a match to find variable declarator with a specific name and with arrow function expression initializer, but it would look somewhat more complex.